pyproject.toml

```diff
@@ -43,7 +43,7 @@ dev = [
   "pytest==7.2.0",
   "pytest-env==0.8.1",
   "pytest-asyncio==0.21.0",
-  "Sphinx==5.3.0",
+  "Sphinx==4.3.0",
```

Comment:
Bumped this down because it had some dependencies for `python < 3.10` that were mutually exclusive with `flake8` & `pytest`. Manual doc generation (and poking around those generated docs) seems to work just fine, though